### PR TITLE
Add Permutations Article.

### LINKED ISSUE #353: Wiki request - rewrite and simplify explanation of Permutations
Original issue: https://github.com/FreeCodeCamp/FreeCodeCamp/issues/3411

https://www.mathsisfun.com/combinatorics/combinations-permutations.html

### PR DESCRIPTION
**Pull request by pjconsidine**
closes #353

### CODE REVIEW

**Comment by raisedadead** [comment]
@pjconsidine Thanks for the PR.

Please change the PR title to something relevant like "Adds Permutations Article". Also take care to do the same for commit messages in future.

Finally the `closes #XXX` tag needs to be out side the \``, for Github to reference this properly.

File: Permutations.md [hunk 1]
```diff
@@ -0,0 +1,10 @@
+# Permutations
+_Permutation_ is a mathematical term for the number of ways a group of objects can be assembled into a set. It's similar to another mathematical term,   _combination_, except for one key difference: with permutations, the order of the itmes in the set makes a difference.
+
+For example, say you were pulling numbers from a hat and counting the different combinations of three numbers. In that case, both `[1,2,3]` and `[3,2,1]` would be a combination of `1`, `2`, and `3` and would count as one combination. 
+
+However, if you were counting permutations of numbers, they would count as two different instances because the numbers in each set are in a different order. 
+
+Permutations can be calculated in one of two ways, depending on whether repeated values are allowed or not. To calculate the number of permutations of `n` objects without repeats, you simply calculate `n!`, or `n * (n-1) * (n-2) ... * 1`. This makes sense, because if you pick one number from the hat and don't put it back before picking the next number, there will be one less number to choose from.
```
Comment (raisedadead) [inline Permutations.md]: - Add additional headings to clarify between the two types of Permuations.
- Can you please add something on the lines of whats a Factorial or point to some article for understanding?

**Comment by raisedadead** [comment]
IMHO This article needs some additional formatting and clearer examples, I feel the original article is much more easier to follow.

Refer : https://www.mathsisfun.com/combinatorics/combinations-permutations.html

**Comment by Rafase282** [comment]
@pjconsidine  please change the title of the pull request to something relevant, if you must put your name and email then put it on the pull request description.

**Comment by Rafase282** [comment]
@pjconsidine the instructions said to sign off your contribution via commit name and pull request title? If so then we must indeed change it. I though we had already added the link on commit guidelines.

I'll take a look at it.

**Comment by pjconsidine** [comment]
Oh sorry. I just looked at the commit here. I was using a different Git client on my machine and didn't realize it was using the signoff as the commit message. I meant the note about not putting the issue number in backticks, since that's how it was in the example.

**Comment by raisedadead** [comment]
@pjconsidine Umm we mention that in backticks, to avoid accidental references. I guess we can clarify that in the guidelines. Sorry for the misinterpretation.

**Comment by raisedadead** [comment]
@pjconsidine IMHO, although it takes effort, the aim here is to make the article easy for anyone, especially newbies to understand the concepts, in as simple way as possible.

Having said that I sincerely appreciate the effort you have taken to write this. And its already generic, and good, but just needs a few touches of maybe images? or maybe a better example like "balls in a jar"

Something that's on the similar lines, of the original article.

For example, many will see `n!` and not get to know that its a factorial. Which needs explanation (even external/internal links to other articles will do).

Please feel free to give your feedbacks on this approach. Also we can be reached via the [Chat room](https://gitter.im/FreeCodeCamp/Wiki) if you would like to discuss

**Pull request merged**